default.json

```diff
@@ -8,8 +8,9 @@
   "packageRules": [
     {
       "matchUpdateTypes": ["minor", "major", "patch", "pin", "pinDigest", "digest"],
-      "minimumReleaseAge": "4 days",
-      "schedule": ["* 0-4 * * 3"]
+      "minimumReleaseAge": "7 days",
+      "schedule": ["* 0-4 * * 3"],
+      "addLabels": ["CI/skip"]
```

Comment:
Should this line not only be in brave-core?

Reply:
should we support this label in crx-packager as well? i saw that uses jenkins too. https://github.com/brave/brave-core-crx-packager/pulls?q=is%3Aopen+is%3Apr. any other jenkins-enabled repos? @mihaiplesa

Reply:
I don't see much issue to add an extra label everywhere, but let me know what you think is the best course here

Reply:
Out of curiosity, why would we skip CI for dependency updates? At least on search we do want to run tests to detect potential breakage.

Reply:
> Out of curiosity, why would we skip CI for dependency updates? At least on search we do want to run tests to detect potential breakage.

@remusao we only want to "skip" CI jobs that run in Jenkins for automated PRs because these are a massive security risk. once someone reviews the PR with write access, they can remove the CI/skip and jenkins will run.

@thypon I do think it would be pretty confusing for people at Brave to see the CI/skip label on repos that don't have jenkins (and maybe mislead them into removing CI for those PRs), so it's better if this can only run in b-c for now.

Reply:
Thanks for clarifying. I indeed got a bit confused so if possible maybe worth adding the label only for Jenkins?

Reply:
Taking care of this in https://github.com/brave/devops/issues/12886 so `addLabels ` line can now be removed here.